src/navsections/sectionBlueprintMiddleware/utils.ts

```diff
@@ -0,0 +1,8 @@
+export const pickPartialRecord = (record: Record<string, any>, keys: string[]) => {
```

Comment:
I think https://lodash.com/docs/4.17.15#pick does the same

Reply:
While the method looks similar, the lodash one supports key paths like `some.nested.property` and looking at their source code, it looks like they do more operations because of that.